api/http/routing.go

```diff
@@ -192,8 +192,6 @@ func NewDeploymentsResourceRoutes(controller *DeploymentsApiHandlers) []*rest.Ro
 			controller.GetDevicesListForDeployment),
 		rest.Get(ApiUrlManagementDeploymentsLog,
 			controller.GetDeploymentLogForDevice),
-		rest.Delete(ApiUrlManagementDeploymentsDeviceId,
```

Comment:
Can we really remove this endpoint with no prior deprecation warning?

Reply:
I personally believe we should remove it, because it is dangerous to keep it there (just like the inventory one).
Technically speaking, we should deprecate it, you are right, but I highly doubt somebody can use that end-point in the wild as it is designed to be called from the decommissioning workflow.